waiter/bin/run-using-minimesos.sh

```diff
@@ -17,6 +17,10 @@ if [ ${EXIT_CODE} -eq 0 ]
 then
     export WAITER_MARATHON=${MINIMESOS_MARATHON%;}
     export WAITER_ZOOKEEPER_CONNECT_STRING=$(echo ${MINIMESOS_ZOOKEEPER} | awk -F/ '{print $3}')
+    export GRAPHITE_SERVER_PORT=5555
+    export JWKS_PORT=8040
+    export JWKS_SERVER_URL="http://127.0.0.1:${JWKS_PORT}/keys"
+    export WAITER_TEST_JWT_ACCESS_TOKEN_URL="http://127.0.0.1:${JWKS_PORT}/get-token?host={HOST}"
```

Comment:
Please move the test script (for k8s and minimesos) changes to separate PRs.

Reply:
Done